titus-api/src/main/java/com/netflix/titus/api/jobmanager/model/job/ContainerResources.java

```diff
@@ -103,43 +122,20 @@ public boolean equals(Object o) {
         if (o == null || getClass() != o.getClass()) {
             return false;
         }
-
         ContainerResources that = (ContainerResources) o;
-
-        if (Double.compare(that.cpu, cpu) != 0) {
-            return false;
-        }
-        if (gpu != that.gpu) {
-            return false;
-        }
-        if (memoryMB != that.memoryMB) {
-            return false;
-        }
-        if (diskMB != that.diskMB) {
-            return false;
-        }
-        if (networkMbps != that.networkMbps) {
-            return false;
-        }
-        if (allocateIP != that.allocateIP) {
-            return false;
-        }
-        return efsMounts != null ? efsMounts.equals(that.efsMounts) : that.efsMounts == null;
+        return Double.compare(that.cpu, cpu) == 0 &&
+                gpu == that.gpu &&
+                memoryMB == that.memoryMB &&
+                diskMB == that.diskMB &&
+                networkMbps == that.networkMbps &&
+                allocateIP == that.allocateIP &&
+                shmMB == that.shmMB &&
+                efsMounts.equals(that.efsMounts);
```

Comment:
This leaves out the `efsMounts != null` guard in the previous code - not sure if that's OK here or not.

Reply:
This was just the intelliJ regenerate to hash/equals. Maybe I chose the wrong format. Will check.